phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexScrutinyToolIT.java

```diff
@@ -551,11 +558,132 @@ public void testMaxOutputRows() throws Exception {
         }
     }
 
+    /**
+     * Tests that the config for max number of output rows is observed
+     */
+    @Test
+    public void testTenantId() throws Exception {
+        boolean doesTableExist = false;
+        try {
+            PhoenixRuntime.getTable(conn, MULTI_TENANT_DATA_TABLE_NAME);
+            doesTableExist = true;
+        } catch (TableNotFoundException e) {
+            doesTableExist = false;
+        }
+
+        // We already run this test, return.
+        if (doesTableExist) {
+            return;
+        }
```

Comment:
This test does not use the test parameters and does not follow the overall design within IndexScrutinyToolIT. I suggest either refactoring it to fit in here or implementing it a separate test class.

Reply:
will refactor